src/components/authoring/course-create.vue

```diff
@@ -205,26 +206,25 @@ export default {
         http.post('courses', {
           ...newCourse,
           authorId: auth.userId,
-          storageId: newId
-          // properties: { enrollment: enrBool }
+          storageId: newId,
+          properties: { enrollment: enrBool }
         }).then(() => {
           // console.log(resp)
           this.$router.push(`/courses/${newCourse.name}/1`)
 
           /* create enrollment for creator */
-          // COMMENTED OUT B/C ENROLLMENT DISABLED (cmc 2021-11-09)
-          // if (enrBool) {
-          //     http.get(`courses/getCourseId?courseName=${newCourse.name}`).
-          //       then( resp => {
-          //         const newEnrollment = {
-          //           courseId: resp.data.courseId,
-          //           studentId: self.auth.userId
-          //         }
-          //         http.patch('enrollments', {
-          //           ...newEnrollment
-          //         }).catch((err) => {console.log(err)})
-          //       })
-          // }
+          if (enrBool) {
+            http.get(`courses/getCourseId?courseName=${newCourse.name}`)
+              .then(resp => {
+                const newEnrollment = {
+                  courseId: resp.data.courseId,
+                  studentId: this.auth.userId
+                }
+                http.patch('enrollments', {
+                  ...newEnrollment
+                }).catch((err) => { console.log(err) })
+              })
+          }
```

Comment:
These methods have been moved to store. There will be a second revision necessary when the `refactors-april-22` is being merged.